packages/react/src/components/ComboBox/ComboBox.js

```diff
@@ -292,6 +292,7 @@ export default class ComboBox extends React.Component {
           clearSelection,
         }) => (
           <ListBox
+            light
```

Comment:
Seems that you are trying to pass along `light` prop to `<ListBox>`, but the syntax here seems to just hard-code `light`. Probably you meant `light={light}`?

Reply:
yeah good catch, and it's already passed into the listbox for all components except dropdown!